consensus/state_sync/helpers.go

```diff
@@ -31,6 +25,7 @@ func (m *stateSync) sendToPeer(msg *anypb.Any, peerId cryptoPocket.Address) erro
 	return nil
 }
 
+// TODO (#352, goku) check if this is needed, if not, remove
```

Comment:
<img width="630" alt="Screenshot 2023-03-16 at 8 55 43 PM" src="https://user-images.githubusercontent.com/1892194/225808882-48160771-fa63-4f7c-9df6-ff19b88860e2.png">

Have you tried asking ChatGPT? I know it's outside the scope of this PR, but it took me just as long to ask ChatGPT as writing the comment :)

Reply:
I removed, thanks.